doc/source/reference/c-api/array.rst

```diff
@@ -997,6 +997,9 @@ argument must be a :c:type:`PyObject *<PyObject>` that can be directly interpret
     called on flexible dtypes. Types that are attached to an array will always
     be sized, hence the array form of this macro not existing.
 
+    ..versionchanged:: 1.18.0
+    For structured datatypes with no fields this function now return False.
```

Comment:
Does this line need to be indented?

typo: `returns`

Reply:
Fixed typo. I indented it the same way as the `..versionadded` in this file so it should be ok.